news/_posts/translation/2020-07-10-do-you-speak-prestashop-june-2020.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title:  "Do you speak PrestaShop? – June 2020 edition"
+subtitle: "Where you'll learn that Esperanto is on the rise"
+date:  2020-07-10 14:30:11
+authors: [ JuneByun ]
+icon: icon-world
+image: /assets/images/2019/01/Build-Crowdin-banner.jpg
+tags: [translation, i18n, l10n]
+---
+
+Contributing to PrestaShop is not only about the code, it's also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! This report will tell you how the translations of the software evolved in May.
+
+![Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
+
+## Project news
+
+
+### First news, Esperanto is on the rise :fire:
+
+I am thrilled to let you know that Esperanto has been on the rise since the past month. From the beginning of May to the end of June, the Esperanto translation project went from 25% to 34% words translated, that is to say a 9% evolution. I wish to take this opportunity to thank mirmal and Milan Kolka (milankolka) for your work! :heart_eyes: 
+
+Many of you dedicate their time to contribute to the PrestaShop projects on Crowdin but cannot validate translations because you don’t have the rights. I thus wanted to tell you to not hesitate in [asking me](https://crowdin.com/profile/june.byun) if you wish to become a proofreader. It would allow you to suggest and approve the strings you and others translate.
```

Comment:
Are you thinking about a project in particular? If yes, I suggest you specify it in this paragraph. Also, note that the proofreader role is something you kind of deserve thanks to a great dedication or a specific involvement, it is not supposed to be granted to anyone who asks for it. A translator is expected to 'apply' for this role.